### PR TITLE
[fix #390] fix menu edit on mac

### LINKED ISSUE #390: Copy/cut/paste undo/redo commands are not available due to lack of top menu shortcuts (breaks text input for OPDS search tags, etc.)
ThoriumNoMenu

This is a regression. The culprit is PR https://github.com/readium/readium-desktop/pull/375

More specifically this code removal:
https://github.com/readium/readium-desktop/pull/375/files#diff-8cfead41d88ad47d44509a8ab0a109ad

Note, the old comment is still there:
https://github.com/readium/readium-desktop/blob/b191a5b350e093d65ee0836813715f3a9bc4aa20/src/main.ts#L130

Removing these command shortcuts break text input / editing.

### PR DESCRIPTION
**Pull request by panaC**
Test on mac-os in dev mode

But why it's 'electron' that appears in top right and not Thorium ?

fixes #390 

I formatted both JSON files

### CODE REVIEW

**Comment by danielweck** [comment]
Regarding the "electron" application name: when testing `r2-testapp-js` on MacO I found it annoying to see the generic Electron icon label when keyboard-alt-tabbing between apps, etc. ... so I created a script to patch the name :)
https://github.com/readium/r2-testapp-js/blob/develop/tools/electronRename.js

File: src/resources/locales/fr.json [hunk 1]
```diff
@@ -1,278 +1,287 @@
 {
-    "app": {
-      "quit": "Quitter l'application"
+  "app": {
+    "quit": "Quitter Thorium",
+    "edit": {
+      "title": "Edition",
```
Comment (danielweck) [inline src/resources/locales/fr.json]: Édition? (accentuated `E`)

**Comment by panaC** [comment]
> Regarding the "electron" application name: when testing `r2-testapp-js` on MacOS I found it annoying to see the generic Electron icon label when keyboard-alt-tabbing between apps, etc. ... so I created a script to patch the name :)
> https://github.com/readium/r2-testapp-js/blob/develop/tools/electronRename.js

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>BuildMachineOSBuild</key>
	<string>16F73</string>
	<key>CFBundleDisplayName</key>
	<string>Electron</string>
	<key>CFBundleExecutable</key>
	<string>Electron</string>
	<key>CFBundleIconFile</key>
	<string>electron.icns</string>
	<key>CFBundleIdentifier</key>
	<string>com.github.electron</string>
	<key>CFBundleInfoDictionaryVersion</key>
	<string>6.0</string>
	<key>CFBundleName</key>
	<string>Electron</string>
	<key>CFBundlePackageType</key>
	<string>APPL</string>
	<key>CFBundleShortVersionString</key>
	<string>3.1.7</string>
	<key>CFBundleVersion</key>
	<string>3.1.7</string>
	<key>DTSDKBuild</key>
	<string>16E185</string>
	<key>DTSDKName</key>
	<string>macosx10.1210.12</string>
	<key>DTXcode</key>
	<string>0833</string>
	<key>DTXcodeBuild</key>
	<string>8E3004b</string>
	<key>LSApplicationCategoryType</key>
	<string>public.app-category.developer-tools</string>
	<key>LSMinimumSystemVersion</key>
	<string>10.9.0</string>
	<key>NSHighResolutionCapable</key>
	<true/>
	<key>NSMainNibFile</key>
	<string>MainMenu</string>
	<key>NSPrincipalClass</key>
	<string>AtomApplication</string>
	<key>NSSupportsAutomaticGraphicsSwitching</key>
	<true/>
</dict>
</plist>
```

When is packaged , it will be replaced by thorium. I think, no ?

**Comment by danielweck** [comment]
The file `./node_modules/electron/dist/Electron.app/Contents/Info.plist` is only used in dev mode (and non-packaged production mode) to launch the Electron runtime with the `./dist/main.js` entry point.
In packaged production mode, `electron-builder` includes a fresh Electron executable into the DMG installer / `*.app` bundle.

**Pull request merged**